Modix.Services/Utilities/ApplyChannelLimitsAttribute.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Discord.Commands;
+using Modix.Data;
+using Modix.Data.Utilities;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Modix.Services.Utilities
+{
+    public class ApplyChannelLimits : PreconditionAttribute
+    {
+        public override Task<PreconditionResult> CheckPermissions(ICommandContext context, CommandInfo command, IServiceProvider map)
+        {
+            using (var db = map.GetService<ModixContext>())
+            {
+                var limits = db.ChannelLimits
+                        .Where(c =>
+                            c.ModuleName.ToUpper() == command.Module.Name.ToUpper() &&
```

Comment:
Shouldn't this be granular down to the group + command name, not just the module?